addons/web_editor/static/src/js/frontend/loader.js

```diff
@@ -38,28 +38,15 @@ async function loadFromTextarea(parent, textarea, options) {
     if (!currentOptions.value || !currentOptions.value.trim()) {
         currentOptions.value = '<p><br><p>';
     }
-    const wysiwyg = await createWysiwyg(parent, Object.assign({
-        template: `<t-dialog><t t-zone="default"/></t-dialog>
-            <div class="d-flex flex-grow-1 flex-column" style="height: 200px">
-                <div class="o_toolbar">
-                    <t t-zone="tools"/>
-                </div>
-                <div class="d-flex flex-grow-1 overflow-auto note-editing-area">
-                    <t t-zone="main"/>
-                </div>
-                <div class="o_debug_zone">
-                    <t t-zone="debug"/>
-                </div>
-            </div>`
-        }, currentOptions));
+    const wysiwyg = await createWysiwyg(parent, currentOptions);
```

Comment:
The `template` key was never used.

Reply:
What ? Comment ça pouvait marcher ?

Reply:
It didn't. Somewhere along the line an `interface` key was added to the `options` object and that's the one that was being used.

Reply:
But where is the template now ?